File: Problem9/palindrome.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False
        else:    
            y = int(0)
            temp = x 
            while temp >= 10:
                y += int(temp % 10)
                y *= int(10)
                temp /= int(10)

            y += int(temp)

            if y == x:
                return True
            else: 
                return False

File: Problem9/palindrome.c
bool isPalindrome(int x){
    if(x < 0)
        return false;
    else{
        unsigned int y = 0;
        unsigned int temp = x; 
        while(temp >= 10){
            y += temp % 10;
            y *= 10;
            temp /= 10;
        }
        y += temp;

        if(y == x)
            return true;
        else
            return false;
    }        
}